frontend/libs/studio-components/src/components/StudioCodeFragment/StudioCodeFragment.tsx

```diff
@@ -3,11 +3,14 @@ import React, { forwardRef } from 'react';
 import cn from 'classnames';
 import classes from './StudioCodeFragment.module.css';
 
-export type StudioCodeFragmentProps = HTMLAttributes<HTMLElement>;
+export type StudioCodeFragmentProps = HTMLAttributes<HTMLElement> & {
+  classname?: string;
+};
```

Comment:
This can be reverted to be as it was before, since the HTMLElement has `className` already. 😊 
```suggestion
export type StudioCodeFragmentProps = HTMLAttributes<HTMLElement>;
```

Reply:
fixed🙂